src/main/java/com/ystrazhko/git/connections/RESTConnectorImpl.java

```diff
@@ -71,10 +71,8 @@ private Object sendRequest(String suffixForUrl, Map<String, String> params, Map<
             System.out.println(response.toString());
             return response.toString();
         } catch (Exception e) {
-            e.printStackTrace();
+            throw new RuntimeException(e.getMessage());

```

Comment:
That's a bad pattern. Throw new exception and put the cause exception in it. new RuntimeException(e). Also, maybe we need to introduce our own exception class for this situation? Do we always need to rethrow the exception? Can any situations except 401 be handled here?